tests/migrations/test_operations.py

```diff
@@ -1855,6 +1855,18 @@ def test_add_constraint(self):
         self.assertEqual(definition[1], [])
         self.assertEqual(definition[2], {'model_name': "Pony", 'constraint': gt_constraint})
 
+    @skipUnlessDBFeature('supports_table_check_constraints')
+    def test_add_or_constraint(self):
+        project_state = self.set_up_test_model("test_addorconstraint", index=True)
```

Comment:
Pretty sure `index=True` is unnecessary here?

Reply:
you're right. I missed it because of copy-paste.